ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,23 @@
 <div class="container-fluid wu-frame">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
Noone has complained about the use of the patternfly styles on the other one place that we use validation (file upload). Admittedly, that one flickers badly.